ocean_provider/utils/provider_fees.py

```diff
@@ -77,6 +80,58 @@ def get_provider_fees(
     return provider_fee
 
 
+def get_provider_fees_or_remote(
```

Comment:
this function is atomicly executed for every input -> leads to user paying providerFee multiple times.
See my previous comment